jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc

```diff
@@ -20,8 +20,9 @@
 === Community
 
 Developers and users alike are welcome to engage the Jetty community.
-We all have day jobs here so don't just ask questions on IRC and frustrated if no one answers right away.
-Stick around to hear an answer, one is likely coming at some point!
+We all have day jobs here so don't just ask questions and get frustrated if 
+no one answers right away. Stick around to hear an answer, one is 
+likely coming at some point!
```

Comment:
I don't think you're supposed to put line breaks in like this.

Reply:
The asciidoc will merge them together.

Don't forget, the asciidoc is supposed to be readable on it's own, before being processed into html/pdf.

Reply:
I was doing this before but told not to by Simone or Chris. The rest of the documentation uses only 1 line per sentence, and to view it in the IDE you use word wrap.